__tests__/app.test.js

```diff
@@ -111,3 +111,61 @@ describe('api/users', () => {
     })  
 })
 
+describe('PATCH api/articles/:article_id', () => {
+    it('should update the article vote property by given amount which is a positive number', () => {
+        const testInput = {inc_votes: 10}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testInput)
+        .expect(200)
+        .then((result) => {
+            expect(result.body.article.votes).toBe(110)
+            })
+        })
+    })
+    it('should update the article vote property by given amount which is a negative number', () => {
+        const testInput = {inc_votes: -10}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testInput)
+        .expect(200)
+        .then((result) => {
+            expect(result.body.article.votes).toBe(90)
+            })
+        })
+    it('should update the article vote property by given amount and return the updated article', () => {
+        const testInput = {inc_votes: -10}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testInput)
+        .expect(200)
+        .then((result) => {
+            expect(result.body.article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 90
+                })
+            })
+    })
+    it('should generate 404 if items not found', () => {
+        return request(app)
+        .patch('/api/articles/20')
+        .expect(404)
+        .then((result) => {
+        expect(result.body).toEqual({ message: 'Item not found' });
+        })
+    }) 
+    it('should generate 400 if user has made a bad request', () => {
+        return request(app)
+        .patch('/api/articles/fridge')
+        .expect(400)
+        .then ((result) => {
+        expect(result.body).toEqual({message: 'Bad request'})
+   
+    })
+})
+
```

Comment:
Please could you add a couple more tests:
 * a test to check if the inc_vote key has a value of 0, what does this do? (check it doesn't change the vote count)
 * a test to check if the vote object doesn't include a key of inc_vote, what should this do?
 * a check to test if the inc_vote key does not have a number as a value (e.g. { inc_vote: "hello" })